src/core/map/sky_roof.rs

```diff
@@ -46,7 +46,8 @@ impl<'w, 's> AssetCommands<'w, 's> {
                 ..Default::default()
             },
             transform: Transform {
-                rotation: Quat::from_euler(EulerRot::XYZ, -PI / 8.0, -PI / 4.0, 0.0),
+                translation: Vec3::new(25.0, 0.0, 0.0),
```

Comment:
Thanks, I also fixed shadows in the latest master :)
Did you add `transform` for a reason? It looks like the map is fully covered even with default transform.

Reply:
I centered the light to the map so that I can make `PROJECTION` smaller -> less aliased shadows.

Reply:
Hm... If I removed explicit `translation` - looks like the map is fully covered with shadows anyway with your projection value.

Reply:
I see. That may be because I also lowered the pitch angle of that light, which makes the projection cover the entire map.

Reply:
Got it, thanks. Probably I can remove explicit `translation` then.